src/protos/flywheels.proto

```diff
@@ -0,0 +1,82 @@
+syntax = "proto3";
+
+/*************************************************/
+/******** Main messages that will be sent ********/
+/*************************************************/
+
+message FlywheelConfig {
+    repeated int32 controlPins = 0;
+    telemMethod telemetry = 1;
+    // Both no telemetry and bidirectional Dshot will not have any telemetry pins
+    // Serial and tachometer will likely only have one pin, but we could support having them communicate on multiple pins
+    repeated int32 telemPins = 2;
+
+    int32 motorKv = 3;
```

Comment:
@ahalekelly Do we want different KV per motor (or motor pair)?

If this like brushed motors where you may want a faster second stage, so you'd spring for a hotter wind?